pkg/api/v2/remove.go

```diff
@@ -0,0 +1,24 @@
+package v2
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	snaputil "github.com/canonical/microk8s-cluster-agent/pkg/snap/util"
+)
+
+// RemoveRequest represents a request to remove a node from the dqlite cluster.
+type RemoveRequest struct {
+	// HostPort is the address of the node to remove.
+	HostPort string `json:"hostPort"`
```

Comment:
nit: Can we find another name for this, HostPort is a kubernetes term and might confuse others.